.github/workflows/cherry-pick-image.yaml

```diff
@@ -0,0 +1,56 @@
+name: cherry-pick-image
+on:
+  workflow_call:
+    inputs:
+      gcr_tag:
+        description: tag to cherry pick
+        type: string
+        required: true
+      source_gcr_url:
+        type: string
+        required: true
+      target_gcr_url:
+        type: string
+        required: true
+  workflow_dispatch:
+      inputs:
+        gcr_tag:
+          description: tag to cherry pick
+          type: string
+          required: true
+        source_gcr_url:
+          description: gcr url to cherry pick issue from
```

Comment:
Not sure what `issue` means here, I was expecting `container` or `image`. Although I don't know how important the description is in the first place, seems like most that are using the UI would already know what these things are.

Reply:
Oh, I definitely meant Image, thanks for catching that.